### PR TITLE
Added GameStar and GamePro

### PR DESCRIPTION
**Pull request by Mailaender**
They are sister magazines from the same publisher and their website layout is pretty much identical. I only found differences between older articles and newer ones, so there are different ways to scrape the correct author.

### CODE REVIEW

File: GameStar GamePro.js [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+		return "magazineArticle";
+	}
+	return false;
+}
+
+function doWeb(doc, url) {
+	if (detectWeb(doc, url) == "magazineArticle") {
+		scrape(doc, url);
+	}
+}
```
Comment (AbeJellinek) [inline GameStar GamePro.js:50]: Could you add a `multiple` case? The "web translator template" generated by Scaffold will give you 95% of the code you need - you'll probably just have to tweak the selectors.

Comment (Mailaender) [reply GameStar GamePro.js:50]: If `multiple` means grabbing search results, then I don't see the purpose of those.

Comment (AbeJellinek) [reply GameStar GamePro.js:50]: A lot of the time, neither do I. But almost every Zotero web translator supports them, unless its target site doesn't have useful search results. That's not the case here, so they should be supported. It'll seriously be ten simple lines of code from the template.

Comment (Mailaender) [reply GameStar GamePro.js:50]: So essentially, you want me to write a parser for stuff like https://www.gamestar.de/suche/?query=grand+theft+auto just like
https://github.com/zotero/translators/blob/2b2de18fad702214f00c292d394c83f644f4e7e5/PC%20Gamer.js#L76-L90
Sadly the HTML/CSS on that site is a lot messier.

Comment (Mailaender) [reply GameStar GamePro.js:66]: I used 
https://github.com/zotero/translators/blob/1d1ce6111ac17a4064c1d8836f6beec54404c32b/PC%20Gamer.js#L76-L90
as a reference and copied the selectors using the Chrome developer tools.

**Comment by Mailaender** [comment]
Also cleared the warnings.

**Comment by AbeJellinek** [comment]
@Mailaender Awesome. Thanks for doing that. I think I found a selector that's a bit more resilient to changes in the site's layout (or different ad blockers, etc.). Pushed a commit. Check it out and let me know what you think.

**Comment by Mailaender** [comment]
Yes, the CSS was a bit messy. Thanks.

**Comment by AbeJellinek** [comment]
Merged! Thanks so much for your contribution.

**Pull request merged**